resources/src/main/java/org/robolectric/manifest/ActivityData.java

```diff
@@ -72,7 +72,7 @@ public boolean isAlwaysRetainTaskState() {
     return getBooleanAttr(withXMLNS(ALWAYS_RETAIN_TASK_STATE), false);
   }
 
-  public boolean isClearTaskOnLaungh() {
+  public boolean isClearTaskOnLaunch() {
```

Comment:
This is a breaking change. Let me know if you prefer to deprecate `isClearTaskOnLaungh()` and introduce a new `isClearTaskOnLaunch()` method instead.